CHANGELOG.md

```diff
@@ -435,7 +435,7 @@ questions.
 * Rework dataflow model and Zed compiler optimizations (#2669)
 * Add initial `explode` operator that can break values from complex fields out into separate records (#2673)
 * Fix an issue where including a particular `time`-typed field in a shaper script caused errors with shaping other fields (#2685)
-* Silently discard duplicate fields when reading NDJSON records, which works around [Suricata bug 4016](https://redmine.openinfosecfoundation.org/issues/4106) (#2691)
+* Silently discard duplicate fields when reading NDJSON records, which works around Suricata bug 4016 (#2691)
```

Comment:
This link to Suricata's flaky bug tracking system kept angering the broken link checker so I just moved the hyperlink into the referenced super issue #2691 and dropped it from this CHANGELOG.